src/core/main.c

```diff
@@ -1406,7 +1406,7 @@ int main(int argc, char *argv[]) {
         bool loaded_policy = false;
         bool arm_reboot_watchdog = false;
         bool queue_default_job = false;
-        bool empty_etc = false;
+        bool first_boot = false;
```

Comment:
<strike>I'm not sure that this goes in the right direction. "empty_etc" refers to a testable condition. "first_boot" also seems to refer to the same condition, but actually we don't know if it's the first boot, or if the previous boot failed before we managed to write some files, or if the previous boot was read-only, or if somebody deleted some files. So the new name is more misleading than the old one.</strike>

OK, enough of the bikeshedding.